tracking_model/mixins.py

```diff
@@ -2,24 +2,33 @@
 
 
 class Tracker(object):
-    def __init__(self, instance, fields=None):
+    def __init__(self, instance, tracking_fields):
         self.instance = instance
         self.newly_created = False
         self.changed = {}
-        self.tracked_fields = {f.attname for f in self.instance._meta.concrete_fields}
-        if fields is not None:
-            self.tracked_fields &= set(fields)
+        self.tracked_fields = tracking_fields
```

Comment:
self.tracked_fields = self.instance.TRACKED_FIELDS (instance will point to class object)